resources/language/resource.language.cs_cs/strings.po

```diff
@@ -0,0 +1,965 @@
+msgctxt "#30191"
+msgid "Do you want to export NFO files for the {} added to 'My List'?"
+msgstr "Chcete exportovat soubory NFO pro {} přidané pro 'Můj seznam'?"
+
+msgctxt "#30192"
+msgid "\r\nThese files are used by the provider information scrapers to integrate movies and tv shows info, useful with bonus episodes or director's cut"
+msgstr "\r\nTyto soubory používají poskytovatelé scraperů pro integraci informací o filmech a seríálech, užitečné s bonusovými díly nebo režisérským střihem"
+
+msgctxt "#30193"
+msgid "Include all information in NFO files (use only with 'Local Information' scraper)"
+msgstr "Zahrnout všechny informace do souborů NFO (použít pouze se scraperem 'Local Information')."
+
+msgctxt "#30194"
+msgid "Limit video stream resolution to"
+msgstr "Omezit rozlišení videa na"
+
+msgctxt "#30195"
+msgid "Export new episodes"
+msgstr "Exportovat nové díly"
+
+msgctxt "#30196"
+msgid "Exclude from auto update"
+msgstr "Vyloučit z automatické aktualizace"
+
+msgctxt "#30197"
+msgid "Include in auto update"
+msgstr "Zahrnout do automatické aktualizace"
+
+msgctxt "#30198"
+msgid "Exporting new episodes"
+msgstr "Export nových dílů"
+
+msgctxt "#30199"
+msgid "Shared library (Kodi MySQL server is required)"
+msgstr "Sdílená knihovna (je vyžadován server Kodi MySQL)"
+
+msgctxt "#30200"
+msgid "Use MySQL shared library database"
+msgstr "Použít databázi sdílených knihoven MySQL"
+
+msgctxt "#30201"
+msgid "Username"
+msgstr "Uživatelské jméno"
+
+msgctxt "#30202"
+msgid "Connection to the MySQL database was successful"
+msgstr "Připojení k databázi MySQL bylo úspěšné"
+
+msgctxt "#30203"
+msgid "Host IP"
+msgstr "Hostitelská IP"
+
+msgctxt "#30204"
+msgid "Port"
+msgstr ""
+
+msgctxt "#30205"
+msgid "Test database connection"
+msgstr "Testovat připojení k databázi"
+
+msgctxt "#30206"
+msgid "ERROR: The MySQL database is not reachable - the local database will be used"
+msgstr "CHYBA: Databáze MySQL není dostupná - použije se lokální databáze"
+
+msgctxt "#30207"
+msgid "Set this device as main auto-updates manager"
+msgstr "Nastavit toto zařízení jako hlavního správce automatických aktualizací"
+
+msgctxt "#30208"
+msgid "Check if this device is the main auto-update manager"
+msgstr "Zkontrolujte, zda je toto zařízení hlavním správcem automatické aktualizace"
+
+msgctxt "#30209"
+msgid "This device now handles the auto-updates of shared-library"
+msgstr "Toto zařízení nyní zpracovává automatické aktualizace sdílené knihovny"
+
+msgctxt "#30210"
+msgid "This device handles the auto-updates of shared-library"
+msgstr "Toto zařízení zpracovává automatické aktualizace sdílené knihovny"
+
+msgctxt "#30211"
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Jiné zařízení zpracovává automatické aktualizace sdílené knihovny"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Neexistuje žádné zařízení, které by zpracovávalo automatické aktualizace sdílené knihovny"
```

Comment:
Using the comma "," here makes it unmanageable for Kodi to display the message, remove it